less/mixins.less

```diff
@@ -1 +1,29 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.flex-center(@flex-direction, @justify-content, @align-items) {
+    display: flex;
```

Comment:
Were you using "" to send in blank values to error out the properties. If so. Clever. But I don't think its good practice. Also I like to use default values in case I forget to call the mixin with all of the params. It won't break less.